Sources/ObjCSupport/Builders/LOKInsetLayoutBuilder.h

```diff
@@ -11,20 +11,52 @@
 #import "LOKLayoutBuilder.h"
 
 @class LOKInsetLayout;
-
+/**
+ A layout builder for @c LOKInsetLayout.
+ */
 @interface LOKInsetLayoutBuilder: NSObject<LOKLayoutBuilder>
 
+/**
+ Creates a @c LOKInsetLayoutBuilder with the given insets and sublayout.
+ @param insets      The @c LOKEdgeInsets for layout.
+ @param sublayout   The layout placed inside the @c LOKInsetLayout.
+ */
 - (nonnull instancetype)initWithInsets:(LOKEdgeInsets)insets around:(nonnull id<LOKLayout>)sublayout;
 + (nonnull instancetype)withInsets:(LOKEdgeInsets)insets around:(nonnull id<LOKLayout>)sublayout;
 
+/**
+ @c LOKInsetLayoutBuilder block for defining how this layout is positioned inside its parent layout.
+ */
 @property (nonatomic, nonnull, readonly) LOKInsetLayoutBuilder * _Nonnull(^alignment)(LOKAlignment * _Nullable);
+
+/**
+ @c LOKInsetLayoutBuilder block for setting flexibility of the @c LOKInsetLayout.
+ */
 @property (nonatomic, nonnull, readonly) LOKInsetLayoutBuilder * _Nonnull(^flexibility)(LOKFlexibility * _Nullable);
+
+/**
+ @c LOKInsetLayoutBuilder block for setting the viewReuseId used by LayoutKit.
+ */
 @property (nonatomic, nonnull, readonly) LOKInsetLayoutBuilder * _Nonnull(^viewReuseId)(NSString * _Nullable);
+
+/**
+ @c LOKInsetLayoutBuilder block for setting the view class for the @c LOKInsetLayout.
+ */
 @property (nonatomic, nonnull, readonly) LOKInsetLayoutBuilder * _Nonnull(^viewClass)(Class _Nullable);
 
+/**
+ Layoutkit configuration block called with created @c LOKView.
+ */
 @property (nonatomic, nonnull, readonly) LOKInsetLayoutBuilder * _Nonnull(^config)( void(^ _Nullable)(LOKView *_Nonnull));
+
+/**
+ @c LOKInsetLayoutBuilder block for setting edge inset (positive) for the @c LOKInsetLayout.
```

Comment:
I have seen cases where people used negative insets especially to position subviews outside of a parent view's bounds. Please remove `(positive)` from the comment here and in all other files.